integration_tests/steps/kpi.py

```diff
@@ -21,6 +21,7 @@
 import ast
 from behave import then
 from math import isclose
+from d3a.constants import DEVICE_PENALTY_RATE
```

Comment:
Added a a constant [parameter](https://github.com/gridsingularity/d3a-interface/blob/master/d3a_interface/constants_limits.py#L61) in d3a-interface lately

Reply:
Why percentage ? I expected that the penalty is just an energy rate based on the energy deficit / surplus of the load/ pv respectively.

Reply:
@faizan2590  since it was clarified by Sarah that device penalty is not a percentage but just an energy rate, I will remove the `PENALTY_PERCENTAGE ` parameter from the d3a-interface.